themes/default/content/blog/project-scoped-stacks-in-self-managed-backend/index.md

```diff
@@ -0,0 +1,57 @@
+
+See [Deciding On a State Backend](/docs/intro/concepts/state/#deciding-on-a-state-backend) for more on how to choose a state backend.
+
+## Introducing Project-Scoped Stacks in Self-Managed Backends
+
+Up until the latest Pulumi release, one difference between the Service backend and self-managed backends is that in the Service, stacks are scoped by organization and project, whereas in self-managed backends there was previously no scoping. In practice, this meant you couldn't have stacks with the same name (like "dev" or "production") across multiple projects in the same self-managed backend. You'd have to use fully unique stack names within the self-managed backend, such as by manually including the project name in the stack name (e.g. "my-project-production" and "another-project-production").
+
+With the latest Pulumi release, stacks created in new or empty self-managed backends are scoped by project by default, so you no longer have to include the project name in the stack name to make it unique within a backend. You can simply create a new project and use simple names for the project's stacks, such as "dev", "staging", "production", etc., and other projects within the backend can use the same simple stack names.
+
+## Upgrading Existing Self-Managed Backends
+
+Existing self-managed backends remain non-scoped until upgraded. You can upgrade an existing self-managed backend using the new `pulumi state upgrade` command. This command will upgrade all stacks in the backend to be scoped by project.
+
+```
+$ pulumi state upgrade
+This will upgrade the current backend to the latest supported version.
+Older versions of Pulumi will not be able to read the new format.
+Are you sure you want to proceed?
+Please confirm that this is what you'd like to do by typing `yes`:
+```
+
+If you've included the project name in your stack names as a way of making the stack name fully unique within a self-managed backend, you can additionally use the `pulumi stack rename` command to clean up your stack names.
+
+## Old Pulumi Versions
+
+Old versions of Pulumi will not be able to see stacks scoped by project. New versions of Pulumi (v3.61.0 or later) will output a warning if an older CLI has created any non-scoped stacks in a self-managed backend that has project-scoped stacks, suggesting to use the `pulumi state upgrade` command to upgrade the stacks.
+
+## Referencing Project-Scoped Stacks
+
+Project-scoped stacks in self-managed backends are scoped under a virtual organization, a constant value ("organization"). This allows for a consistent stack identity format across both Service and self-managed backends, e.g. with self-managed backends you can refer to a fully-qualified stack with `organization/my-project/my-stack`, or using the shorthands `organization/my-stack` or `my-stack` (depending on context) just like with the Service backend.
```

Comment:
Suggest a code example of "this" becomes "this". Single TS example should be sufficient.